charts/agent-operator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: grafana-agent-operator
 description: A Helm chart for Grafana Agent Operator
 type: application
-version: 0.1.13
+version: 0.2.0
```

Comment:
just confirming that these changes are backwards-compatible? (thanks for bumping the minor version, we should have been following semver anyways...)